Formula/pdmenu.rb

```diff
@@ -0,0 +1,34 @@
+class Pdmenu < Formula
+  desc "Full screen menuing system for Unix"
+  homepage "https://joeyh.name/code/pdmenu/"
+  url "http://sources.buildroot.net/pdmenu/pdmenu-1.3.6.tar.gz"
+  sha256 "dedd2a4a5719278b5e49041161990c2f20b5909818837542aaca01819f2c14eb"
+  license "GPL-2.0-only"
+
+  head "git://git.joeyh.name/pdmenu"
+
+  depends_on "gettext" => :build
+  depends_on "s-lang" => :build
+
+  def install
+    ENV.append "CC", "-I#{HOMEBREW_PREFIX}/include"
+    ENV.append "SLANG_H_LOC", "#{HOMEBREW_PREFIX}/include/slang.h"
+    ENV.append "SLANG_LIB_LOC", "#{HOMEBREW_PREFIX}/lib"
+    ENV.append "INSTALL", "#{HOMEBREW_PREFIX}/bin/ginstall"
```

Comment:
Does this not work with native macOS `install`?

Reply:
no

Reply:
Are you sure? Because `ginstall` actually doesn't exist in this build.

Reply:
Yes I'm sure. without `coreutils` dep (or using `ginstall`) it fails. With `coreutils`, it succeeds.

Reply:
But you don't have `coreutils` here. So `ginstall` can't be used here, because it doesn't exist.